en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -256,6 +249,50 @@ Management Policies** section.
         </tbody>
         </table>
         
+## Adding SMS notification templates
+
+The email notification templates are stored in the `IS_HOME>/repository/conf/email/email-admin-config.xml` 
+file and they can be edited using the Management Console.
+
+!!! tip
+    The **AccountConfirmation** template is used to send email notifications.You can edit and 
+    customize the email template. For more information, 
+    see [Customizing Automated Emails](../../learn/customizing-automated-emails).
+    
+The templates for SMS notifications are stored in the registry. Follow the steps below to add
+SMS notification templates.
+
+!!! Warning
+    All the configurations are case sensitive. Changing the case may break the functionality and 
+    cause errors.
+
+1. Log in to the Management Console and click `Main> Registry> Browse`.
+
+2. On the **tree view** tab, click `system -> config ->identity`.
+
+3. Click **Add Collection**, enter `sms` as the **Name**, and then click **Add**.
+    ![add-sms-collection](../assets/img/learn/self-registration/adding-sms-templates/add-sms-collection.png)
+    
+4. Click `sms` and then click **Add collection**. Enter `accountconfirmation` as the **Name. 
+This will create another directory inside the sms directory.
+    ![add-accountconfirmation-collection](../assets/img/learn/self-registration/adding-sms-templates/add-accountconfirmation-collection.png)
+    
+5. Click on **Add Resource**.
```

Comment:
The content after "Name" should not be in Bold. Only "Add Resource" in step 5 should be bold.

Reply:
Fixed: 933ce08ecb17d0941395f9188251ca65508db919